src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -34,7 +33,7 @@ const ItemUser = ({photo, username, followed, onPress, userid}) => {
         style={[
           styles.containerCard,
           {
-            backgroundColor: isSelect ? COLORS.holytosca15percent : COLORS.white
+            backgroundColor: isSelect ? '#00ADB526' : 'white'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hardcoded color values could be a maintainability issue. It's better to use a centralized place for managing colors, so if you need to change the color scheme in the future, you only need to change it in one place. Consider creating a `colors.js` file and import the colors from there.
```diff
- backgroundColor: isSelect ? '#00ADB526' : 'white'
+ backgroundColor: isSelect ? colors.selectionGreen : colors.white
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->